src/main/java/seedu/address/commons/events/model/EventManagerChangedEvent.java

```diff
@@ -0,0 +1,19 @@
+package seedu.address.commons.events.model;
+
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.model.ReadOnlyEventManager;
+
+/** Indicates the EventManager in the model has changed*/
+public class EventManagerChangedEvent extends BaseEvent {
+
+    public final ReadOnlyEventManager data;
+
```

Comment:
Does this attribute need to be public? Can this be private instead?

As a rule of thumb, always try to have attributes as private. Ignore the comment if I am missing anything obvious :p .